src/azure-cli-core/azure/cli/core/cloud.py

```diff
@@ -127,6 +129,59 @@ def __getattribute__(self, name):
         return val
 
 
+def get_ossrdbms_resource_id(cloud_name):
```

Comment:
please start with "_" for all methods where are private to the current module, and do it across

Reply:
done, thanks